packages/api-explorer/src/components/JsonViewer/index.jsx

```diff
@@ -16,6 +16,7 @@ const collapseButtonStyle = {
 
 export default function JsonViewer({ schema, missingMessage }) {
   const [isCollapsed, setIsCollapsed] = useState(true)
+  const message = missingMessage ? <FormattedMessage id={missingMessage} defaultValue={'missing schema'} /> : null
```

Comment:
Why the if? The message should be always visible. missingMessage is required, so this if is not useful

Reply:
true